src/project-css/app/Helpers/CollectionHelper.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * @author Ajay Krishna Teja Kavur
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Helpers;
+
+use Illuminate\Support\Facades\Storage;
+use App\Models\Collection;
+
+class CollectionHelper {
+    /**
+     * Collection Helper
+     *
+     * These are various functions that help with creating and modifying 
+     * collections.
+     *
+     */
+
+     public function create($data) {
```

Comment:
Needs documentation, what is expected in data and what is the expected return function.  what else is being done in this function?